ProfFile.hs

```diff
@@ -63,40 +63,71 @@ parseLine format s =
         _ -> Left $ "Malformed .prof file line:\n" ++ s
     IncludesSources ->
       case words s of
-        (costCentre:module_:_:no:entries:indTime:indAlloc:inhTime:inhAlloc:other) ->
-          parse' costCentre module_ no entries indTime indAlloc inhTime inhAlloc other
+        (costCentre:module_:rest) | (no:entries:indTime:indAlloc:inhTime:inhAlloc:other) <- dropSRC rest ->
```

Comment:
Two things I do not understand:

* Is this SRC column present only in .prof files produced by new GHCs, and if yes, which version introduced it?
* If yes, will this parser work only with that newer version?

Ideally we'd have it work with both versions.